packages/lesswrong/lib/instanceSettings.ts

```diff
@@ -269,3 +269,5 @@ export const isBotSiteSetting = new PublicInstanceSetting<boolean>('botSite.isBo
 export const aboutPostIdSetting = new PublicInstanceSetting<string>('aboutPostId', 'bJ2haLkcGeLtTWaD5', "warning") // Post ID for the /about route
 
 export const anthropicApiKey = new PublicInstanceSetting<string>('anthropic.claudeTestKey', "LessWrong", "optional")
+
+export const hyperbolicApiKey = new PublicInstanceSetting<string>('hyperbolic.apiKey', "", "optional")
```

Comment:
tempted to say that not implementing openrouter at this point might have been a mistake, but :shrug:

Reply:
I could change towards openrouter. The model wasn't available yesterday on openrouter